ci/testnet-automation.sh

```diff
@@ -28,7 +29,11 @@ launchTestnet() {
   net/init-metrics.sh -e
 
   echo --- start "$nodeCount" node test
-  net/net.sh start -o noValidatorSanity -S solana_*.snap
+  if [[ -n $PREBUILT ]]; then
```

Comment:
Can we use a more descriptive name here: PREBUILT -> USE_PREBUILT_CHANNEL_TARBALL perhaps

Reply:
👍